server/controllers/authorController.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable require-jsdoc */
+import models from '../db/models';
+import util from '../helpers/utilities';
+
+class AuthorController {
+  static async addAuthor(req, res) {
+    const { firstName, middleName, lastName } = req.body;
+    try {
+      const author = await models.Authors.create({
+        firstName,
+        middleName,
+        lastName
+      });
+
+      const authorObj = {};
+      authorObj.authorId = author.id;
+      authorObj.firstName = author.firstName;
+      // Get middleName only when author has one
+      if(author.middleName.length >= 1) {
+        authorObj.middleName = author.middleName;
```

Comment:
Why do you need this check?
It can be undefined if not found

Reply:
For authors that don't have a middleName, the check is meant to return only their firstName and lastName and avoids returning middleName as `null`. 
I've updated the code with the implementation you suggested. 
Thank you.

Reply:
And you understand the new implementation right? It also keeps the code cleaner

Reply:
Yes it do.